### ISSUE TITLE
Meanings of provided weights

### ISSUE THREAD

**Issue by Rtakaha** [event-driven-robotics/hpe-core#103]
Hi, thank you very much for sharing your great work!

I have a question about the weights shared [here](https://github.com/event-driven-robotics/hpe-core/tree/main/example/movenet/models).

There are several trained weights, but what are the differences? It will be great if you give a bit more explanation for each file.

For example, I can imagine that `dhp19_allcams_e33_valacc0.87996.pth` is the weights trained with all cameras of DHP19, with validation accuracy of 0.87996, but I am not sure what `e33` means.

Also, I am not sure about the differences between `pre-trained` and `only-trained`.
`only-trained` is trained with RGB and `pre-trained` is trained with EROS-like frame?

Thank you!

**Comment by arrenglover** [comment]
@gaurvigoyal - can you help here?

**Comment by gaurvigoyal** [comment]
Hi!

These are the different models that were trained for the various experiments stated in our paper "MoveEnet: Online High-Frequency Human Pose Estimation With an Event Camera". 
 
For the models trained on DHP19 dataset, there nomenclature is consistently: 
<<dataset_name>>_<<training_cameras>>_<<number_of_epochs_trained>>_<<validation_accuracy_while_training>>.pth

The others are as follows: 
"[e97_valacc0.81209.pth](https://github.com/event-driven-robotics/hpe-core/blob/main/example/movenet/models/e97_valacc0.81209.pth)" is the final model that is essentially MoveEnet. It is pre-trained on MPII (EROS-like) and finetuned on event-Human 3.6m.

[mpii_pre-trained.pth](https://github.com/event-driven-robotics/hpe-core/blob/main/example/movenet/models/mpii_pre-trained.pth) is the model trained only on MPII dataset (EROS-like). 

[h36m_only-trained.pth](https://github.com/event-driven-robotics/hpe-core/blob/main/example/movenet/models/h36m_only-trained.pth) is the model trained only on event-Human 3.6m without any pre-training.

"[h36m_finetuned.pth](https://github.com/event-driven-robotics/hpe-core/blob/main/example/movenet/models/h36m_finetuned.pth)" is the model pre-trained with MPII (pre-training) and finetuned on the first version of event-Human 3.6m which was not cropped. This version of the dataset was not make publicly available. We will consider removing it in the future. 
 
 If there are more questions, do not hesitate to ask.

**Comment by Rtakaha** [comment]
Hi @gaurvigoyal,

Thank you very much for your explanation! Now I know the meanings.

I have no more issues for now, so I am closing this. Thank you.
